tests/PatternAssertions/BasePatternAssert.php

```diff
@@ -113,7 +113,11 @@ protected function assertElementText($expected, string $selector, Crawler $crawl
     }
     $this->assertElementExists($selector, $crawler);
     $element = $crawler->filter($selector);
-    self::assertEquals($expected, trim($element->text()));
+    $actual = trim($element->text());
+    self::assertEquals($expected, $actual, \sprintf(
+      'Expected value "%s" is not equal actual value "%s" in the selector "%s".',
+      $expected, $actual, $selector
+    ));
```

Comment:
Hope additional message will simplify test debugging - otherwise it is very difficult to understand where issue is.

Reply:
Great stuff, this should have been from the beginning there to be honest.